tokenomics/etherdai.md

```diff
@@ -0,0 +1,33 @@
+# EtherDai
+
+EtherDai is a synthetic Ether token backed by liquid staked Ether tokens and debt issued by Maker. It is pegged to the value of Ether. EtherDai is one of the central products of the Endgame Plan. 
+
+{% hint style="warning" %}
+This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date.
+{% endhint %}
+
+## Endgame Interactions
+
+Maker Core onboards EtherDai as one or more Maker Vaults, allowing users to generate DAI using EtherDAI as collateral. Owners of EtherDAI vaults receive significant SubDAO token emissions through a yield farming system.
```

Comment:
Maker Vaults -> Maker Vault types maybe? Also, not sure if we want the word significant in "significant SubDAO emissions"

Reply:
Also maybe a link to the SubDAO tokenomics page?